app/uk/gov/hmrc/traderservices/views/PrivateBetaStartPageView.scala.html

```diff
@@ -0,0 +1,129 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.traderservices.controllers.routes
+
+@this(
+        govukLayout: uk.gov.hmrc.traderservices.views.html.templates.GovukLayoutWrapper,
+        html: uk.gov.hmrc.traderservices.views.components.html
+)
+
+@(postAction: Call)(implicit request: Request[_], messages: Messages)
+
+@govukLayout(
+  pageTitle = pageTitle("view.beta.title")) {
+
+  @html.h1(messages("view.beta.heading"), classes = "govuk-heading-l")
+  @html.p { @messages("view.beta.lede") }
+
+  <aside>
+    <nav class="contents-list">
+      <h2 class="contents-list__title" id="contents">@messages("view.beta.contents")</h2>
```

Comment:
Can see "Contents" is a H2 but shows like this on the page 
<img width="606" alt="image" src="https://user-images.githubusercontent.com/55190161/109536609-d835b600-7ab5-11eb-8792-4c2dece09970.png">

Reply:
Yes, this is the correct GDS pattern, eg. https://www.gov.uk/eori.
It was discussed with Meg and she was happy with it.